shared-lib/flowy-grid-data-model/src/revision/grid_rev.rs

```diff
@@ -185,15 +187,20 @@ impl FieldRevision {
     }
 }
 
+/// The macro, [impl_type_option] will implement the [TypeOptionDataEntry] for the type that
+/// supports serde trait and TryInto<Bytes> trait.
 pub trait TypeOptionDataEntry {
     fn json_str(&self) -> String;
     fn protobuf_bytes(&self) -> Bytes;
 }
 
+/// The macro, [impl_type_option] will implement the [TypeOptionDataDeserializer] for the type that
+/// supports serde trait and TryFrom<Bytes> trait.
 pub trait TypeOptionDataDeserializer {
     fn from_json_str(s: &str) -> Self;
     fn from_protobuf_bytes(bytes: Bytes) -> Self;
 }
+
 pub type FieldId = String;
 #[derive(Debug, Clone, Default, PartialEq, Eq, Serialize, Deserialize)]
 pub struct RowRevision {
```

Comment:
We can't see it here, but lines 208-210 might have the wrong doc prefix.  /// is used, I think it should be //! (but I'm not sure)